dashboard/app/models/user.rb

```diff
@@ -733,9 +733,16 @@ def primary_script
   end
 
   def needs_to_backfill_user_scripts?
-    user_scripts.empty? && !user_levels.empty?
+    user_scripts.empty? &&
+      !user_levels.empty? &&
+      # Backfill only applies to users created before UserScript model was introduced.
+      created_at < Date.new(2014, 9, 11)
```

Comment:
Since the PR date is when it was merged to staging, do you know when this change was pushed to prod (in UTC)? It wouldn't cost too much to make the date a bit later to be safe (this is pretty minor especially if we intend to run the backfill for all users).